config/initializers/inflections.rb

```diff
@@ -14,3 +14,8 @@
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
 #   inflect.acronym 'RESTful'
 # end
+
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym('SplitterZ')
```

Comment:
It didn't make sense to change the file name for these 2 programs to me, since logically they couldn't be considered words.  Instead by making them an inflector zeitwerk will use this definied camelization instead of needing the underscores.  I can do this for any of the other programs if you think that would be a better idea than renaming the files

Reply:
Can we add a comment to each of these programs' files explaining that? I'm afraid we'll add a program later that needs this, forget about it, and be confused for hours or days about why it doesn't work the same way the others do.

Reply:
I can totally see that happening as well, will do.